anesthetic/samples.py

```diff
@@ -345,6 +347,64 @@ def plot_2d(self, axes, *args, **kwargs):
 
         return fig, axes
 
+    def credibility_interval(self, key, level=0.68, method="iso-probability"):
+        """Compute the marginalised credibility interval.
+
+        Also referred to as confidence interval or iso-probability contours.
+        Also options to compute upper or lower limits instead.
+
+        Note: This is based on the discrete set of points and weights without
+        interpolation. Discretization error is determined by the (normalized)
+        weights, i.e. is small for large sample sizes with small weights.
+
+        Parameters
+        ----------
+        key: str
+            Variable name
+        level: float
+            confidence level. Default 0.68
+        method: str
+            Whether to calculate iso-probability interval (interval with
+            same probability density at each end, "iso-probability", this
+            is also the smallest interval containing `level` fraction of
+            samples), or whether to compute one-sided limits "lower-limit" /
+            "upper-limit" for which `level` fraction of the samples lie
+            above / below the limit.
+
+        Returns
+        -------
+        limit: tuple or float
+            Tuple (iso-probability interval) or float (upper/lower limit)
+        """
+        assert level < 1, "Level >= 1 is not allowed!"
```

Comment:
Could you change the assertions to `raise ValueError`, which is consistent with the rest of the code

Reply:
Done